packages/share_plus/share_plus/example/integration_test/share_plus_test.dart

```diff
@@ -14,4 +14,8 @@ void main() {
   testWidgets('Can launch share', (WidgetTester tester) async {
     expect(Share.share('message', subject: 'title'), completes);
   });
+
+  testWidgets('Test', (WidgetTester tester) async {
```

Comment:
Can you find a better test name? `Test` is not good.

`Can launch shareWithResult` for example

Reply:
Sure